### ISSUE TITLE
Segfaults for tuples-of-arrays with type declaration

### ISSUE THREAD

**Issue by benharsh** [chapel-lang/chapel#7716]
### Summary of Problem
A segfaulting program can be written by declaring a tuple of arrays:
```chpl
var A : [1..10] int;
var q = (A, A, A); // this is fine
var t : 3*A.type; // segfault, nil-dereference, or other errors
```

**Associated Future Test(s):**
[`test/arrays/vass/tuple-with-array-1.chpl`](
           https://github.com/chapel-lang/chapel/blob/master/test/arrays/vass/tuple-with-array-1.chpl
          )

**Comment by cassella** [comment]
This test's behavior is now a runtime error:

```
nil
test/arrays/vass/tuple-with-array-1.chpl:9: error: default initialization of tuple containing array or domain type not yet implemented
```

This may be a dup of #17050, remaining after #15759 / #13149 / #7628.

**Comment by benharsh** [comment]
In https://github.com/chapel-lang/chapel/pull/21297 we concluded that the test in this issue is a duplicate of https://github.com/chapel-lang/chapel/issues/17050 and removed the ``tuple-with-array-1`` test.
